packages/ds-healthcare-gov/src/types/Header/index.d.ts

```diff
@@ -10,33 +10,6 @@ interface Link {
   onClick?: (...args: any[]) => any;
 }
 
-export interface HeaderProps {
-  className?: string;
-  initialLanguage?: Language;
-  switchLocaleLink?: string;
-  loggedIn?: boolean;
-  hideLoginLink?: boolean;
-  hideLogoutLink?: boolean;
-  hideLanguageSwitch?: boolean;
-  firstName?: React.ReactNode;
-  subpath?: string;
-  primaryDomain?: string;
-  skipNavHref?: string;
-  onSkipNavClick?: (...args: any[]) => any;
-  deConsumer?: boolean;
-  deBrokerName?: string;
-  links?: Link[];
-  submenuTop?: React.ReactNode;
-  submenuBottom?: React.ReactNode;
-  headerBottom?: React.ReactNode;
-}
-
-export class Header extends React.Component<HeaderProps, any> {
-  render(): JSX.Element;
-}
-
-export default Header;
```

Comment:
Does this whole file still need to exist? If yes, does the `defaultMenuLinks` definition need to be updated to match the updated params in defaultMenuLinks.ts?

Reply:
Nope, it should be deleted